conf/messages

```diff
@@ -385,12 +385,12 @@ client_agent.notice.line2=The annual tax summary will provide a breakdown of how
 
 # Service unavailable page
 #============================================================
-ats.service_unavailable.title=Sorry there is a problem with the service
-ats.service_unavailable.heading=Sorry there is a problem with the service
+ats.service_unavailable.title=Sorry, the service is unavailable
+ats.service_unavailable.heading=Sorry, the service is unavailable
```

Comment:
So the ticket here is about moving these messages to the application.conf file so we can quickly modify them. Give me a shout on Monday and I can show you the work required for this.